setup-ap/action.yml

```diff
@@ -14,12 +14,26 @@ outputs:
 runs:
   using: composite
   steps:
+    - name: Get AP Tag
+      id: getAPtag
+      shell: bash
+      run: |
+        if [[ "${{inputs.version}}" == "latest" ]]; then
+          echo "releasetag=$(curl -s https://api.github.com/repos/ArchipelagoMW/Archipelago/releases/latest | jq '.tag_name' | sed 's/\"//g')" >> $GITHUB_OUTPUT
+        fi
+        if [[ "${{inputs.version}}" != "latest" ]]; then
+          echo "releasetag=${{ inputs.version }}" >> $GITHUB_OUTPUT
+        fi
+    - name: Echo chosen releasetag
```

Comment:
This is debug and shouldn't be in here anymore. If you need that info later on you can run the action with debug enabled and it'll show you the various input/outputs for steps